src/webgpu/api/validation/render_pipeline/inter_stage.spec.ts

```diff
@@ -262,11 +262,13 @@ g.test('max_components_count,output')
   )
   .params(u =>
     u.combine('isAsync', [false, true]).combineWithParams([
-      // Number of user-defined output scalar components in test shader = device.limits.maxInterStageShaderComponents + numScalarDelta.
+      // Number of user-defined output scalar components in test shader =
+      //     Math.floor((device.limits.maxInterStageShaderComponents + numScalarDelta) / 4) * 4.
       { numScalarDelta: 0, topology: 'triangle-list', _success: true },
       { numScalarDelta: 1, topology: 'triangle-list', _success: false },
       { numScalarDelta: 0, topology: 'point-list', _success: false },
-      { numScalarDelta: -1, topology: 'point-list', _success: true },
+      { numScalarDelta: -1, topology: 'point-list', _success: false },
```

Comment:
Maybe numScalarDelta -3 with point-list is something that we should test as well?